### PR TITLE
Fix windows service management

### LINKED ISSUE #33: Windows sensu service DSC error for starting
## Description of problem

- What did you do? Stopped the Windows sensu-client service and ran puppet
- What happened? The puppet run threw a DSC error about creation of the service as it already existed and skipped starting the service
- What did you expect to happen? Puppet to start the Windows sensu-client service which does happen on Linux clients correctly
- How can someone reproduce the problem? Stop the Windows sensu-client service and run puppet

## Command used and debugging output

- What does your puppet config look like (including any hiera config)
- Is this a masterless or master based puppet setup? Master based

## Platform and version information

Your OS: Redhat 7.8 (Maipo)
Your Ruby version: ruby 2.0.0.648-36.el7.x86_64
Your version of Puppet: 5.5.20 open source
Your version of Sensu: 1.8.0.1 (Windows client)
Your version of this module: 3.4.1
DSC module version: 1.9.3

## Anything else to add that you think will be helpful?
I have attached the Puppet run below but the main part of the error is:

`Error: Could not set 'present' on ensure: PowerShell DSC resource MSFT_ServiceResource  failed to execute Set-TargetResource functionality with error message: The service 'sensu-client' to create alre
ady exists with path 'c:\opt\sensu\bin\sensu-client.exe'  (file: /etc/puppetlabs/code/environments/test/modules/external/sensuclassic/manifests/client.pp, line: 75)
Error: Could not set 'present' on ensure: PowerShell DSC resource MSFT_ServiceResource  failed to execute Set-TargetResource functionality with error message: The service 'sensu-client' to create alre
ady exists with path 'c:\opt\sensu\bin\sensu-client.exe'  (file: /etc/puppetlabs/code/environments/test/modules/external/sensuclassic/manifests/client.pp, line: 75)
Wrapped exception:
PowerShell DSC resource MSFT_ServiceResource  failed to execute Set-TargetResource functionality with error message: The service 'sensu-client' to create already exists with path 'c:\opt\sensu\bin\sen
su-client.exe'`

[Puppet_Run_Service_Start_Issue.txt](https://github.com/sensu/puppet-module-sensuclassic/files/4917907/Puppet_Run_Service_Start_Issue.txt)

### PR DESCRIPTION
**Pull request by treydock**
# Pull Request Checklist

## Description

Replace dsc_service with Exec that installs the sensu-client service. All parameter behaviors are the same, the only change is how the service gets installed.

## Related Issue

Fixes #33

## Motivation and Context

This more closely matches the steps documented by Sensu for installing the client service on Windows.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# @summary Base Sensu class
+# n@summary Base Sensu class
```
Comment (ghoneycutt) [inline manifests/init.pp]: This looks like a typo

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

File: manifests/init.pp (hunk 1 above)
Comment (treydock) [reply manifests/init.pp]: Yep, vim and multi tasking didn't mix. I fixed this.

**Review by ghoneycutt — APPROVED** [review]

**Comment by StuartMcManus** [comment]
Cheers @treydock will try out the change this week to test and will advise. Shame about moving to an exec but not surprising about DSC being an issue really as I can see a gotcha you need to be aware of when using user rights assignment with multiple policies.

**Comment by StuartMcManus** [comment]
@treydock @ghoneycutt I have tested out the change and looks good. It creates it correctly with identity info passed to it and starts the service when down without any issues. Cheers.

Do you know when this might make it into PuppetForge as a version?

**Comment by treydock** [comment]
@StuartMcManus This was released to PuppetForge as v3.6.2.

**Pull request merged**